From ccc165063c128713e5eac52ace050e38ce72de02 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Fri, 28 Jan 2011 00:18:47 -0800
Subject: [PATCH:smproxy] Explicitly depend on libICE since ICE functions are called directly

Without this fix, once Bug 33390 removes "Requires: ice" from the sm.pc
file, smproxy fails to build with undefined symbol errors for
IceSetErrorHandler, IceConnectionNumber, and IceProcessMessages.

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
Reviewed-by: Julien Cristau <jcristau@debian.org>
---
 configure.ac |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index 0daa9f2..98687c3 100644
--- a/configure.ac
+++ b/configure.ac
@@ -38,7 +38,7 @@ XORG_DEFAULT_OPTIONS
 AC_CHECK_FUNCS([mkstemp])
 
 # Checks for pkg-config packages
-PKG_CHECK_MODULES(SMPROXY, sm xt xmuu)
+PKG_CHECK_MODULES(SMPROXY, [sm ice xt xmuu])
 
 AC_CONFIG_FILES([
 	Makefile
-- 
1.7.3.2

